.gitignore

```diff
@@ -62,7 +62,10 @@ dist
 !.elasticbeanstalk/*.cfg.yml
 !.elasticbeanstalk/*.global.yml
 
-# AWS DOcker build
+# AWS Docker build
```

Comment:
![Thank You](https://31.media.tumblr.com/19525737b5bfd6a07f8e9adb1177433e/tumblr_inline_my378uYZ8y1srt7rx.gif)